src/fsharp/FSharp.Core/array.fs

```diff
@@ -120,8 +121,8 @@ namespace Microsoft.FSharp.Collections
             
         [<CompiledName("Replicate")>]
         let replicate count x = 
-            if count < 0 then invalidArg "count" (SR.GetString(SR.inputMustBeNonNegative))
-            let arr = (Microsoft.FSharp.Primitives.Basics.Array.zeroCreateUnchecked count : 'T array)  
+            if count < 0 then invalidArg "count" LanguagePrimitives.ErrorStrings.InputMustBeNonNegativeString
```

Comment:
Actually IIRC it's hard to fix - amking those "inlined" values into functions involves adding surface area to FSharp.Core.  There was some reason we pulled out of fixing it though I'm not sure why.

Is there any particular reason you want to use the direct string?